bot.py

```diff
@@ -2,25 +2,27 @@
 import json
 import os
 
-CHANNEL = 'musicreactions'
-USERNAME = 'Discobear'
 ICON = ":guitar:"
 
 class Bot(object):
     def __init__(self):
         self._client = SlackClient(self._token())
-        self._channel = CHANNEL
         self._icon = ICON
-        self._username = USERNAME
         self._client.rtm_connect()
 
     def _token(self):
         return os.environ['SLACKBOT_TOKEN']
+    
+    def _channel(self):
+        return os.environ['CHANNEL']
```

Comment:
Good idea!
Please add these to the sample Usage in the README file. 
Also, maybe add a default? (`os.environ.get('CHANNEL', 'musicreactions')`), may be nice for people who already set this up.